records/2022-11-22.security.blazer.md

```diff
@@ -0,0 +1,45 @@
+# Introducing Blazer and its Implementation for Notify
+
+Date: 2022-11-22
+
+## Status
+
+**DRAFT**
+
+## Context
+In order to have a thorough understanding of data in our system, Notify devs wanted a collaborative tool which would allow them to explore the database. The tool would allow us to store queries and share them between developers. An added bonus would be if the tool graphed results in a meaningful manner. The goal of this ADR is to explain the implementation of the tool that was chosen for exploring the database.
+
+## Blazer
+
+Going through a list of products that are available and meet our main requirement of collaboration and graphing we decided on https://github.com/ankane/blazer
+
+### Implementation
+
+Introducing blazer into our system requires us to do the following:
+
+1. Blazer needs access to the main Notify DB
+2. Blazer needs to be secure and auditable
```

Comment:
Where could I find the audits if I want to know who executed which queries?

Reply:
added this under monitoring